libs/islamic-website/ui-kit/src/lib/members-container/members-container.tsx

```diff
@@ -87,7 +87,7 @@ export function MembersContainer({
           })}
         </Swiper>
       ) : (
-        <div className="mt-[24px] grid grid-cols-2 gap-[32px] md:mt-[28px] md:grid-cols-4 lg:mt-[32px]">
+        <div className="mt-[24px] grid grid-cols-2 gap-[32px] md:mt-[28px] lg:mt-[32px] lg:grid-cols-4">
```

Comment:
There is not a single place in the layouts where the cards are 4 in a row

```suggestion
        <div className="mt-[24px] grid grid-cols-2 gap-[32px] md:mt-[28px] lg:mt-[32px] md:grid-cols-3">
```